scalyr_agent/copying_manager/worker.py

```diff
@@ -1215,7 +1215,83 @@ def create_shared_object_manager(worker_class, worker_proxy_class):
     """
 
     class _SharedObjectManager(multiprocessing.managers.SyncManager):
-        pass
+        # the PID of the parent process.
+        pid = None
```

Comment:
Should be better to call it ``ppid`` or similar since you may think it refers to the actual worker / child process pid.